### PR TITLE
Share URL controller migration as component

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Share URL controller migration as a component. 
Also migrated focus-me directive as it was used by this component

## Motivation and Context
Example of ng-controller migration
Angular migration

## How Has This Been Tested?
Locally and [stage-1]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/partials/billing/share-url-popover.html [hunk 1]
```diff
@@ -1,12 +1 @@
...
-    <streamline-icon name="close" width="10" height="10"></streamline-icon>
-  </button>
-
-  <div class="aligner mt-3">
-    <input class="form-control copy-text-box w-100" type="text" readonly="readonly" value="{{getUrl()}}" ng-focus="onTextFocus($event)" focus-me="true"/>
-    <button type="button" id="copyUrlButton" class="btn btn-primary ml-2" ng-click="copyToClipboard(getUrl())" aria-label="Copy">Copy</button>
-  </div>
-
-</div>
+<share-url (on-close)="dismiss()"></share-url>
```
Comment (ezequielc) [inline src/partials/billing/share-url-popover.html:1]: @alex-deaconu This is being used inside a popover directive and calls the dismiss() from the parent scope. That is why I needed to trigger the dismiss from here and not form inside the component. Thanks

File: src/app/components/focus-me/focus-me.directive.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Directive, ElementRef, Input } from '@angular/core';
+
+@Directive({
+  selector: '[focusMe]'
+})
+export class FocusMeDirective {
```
Comment (ezequielc) [inline src/app/components/focus-me/focus-me.directive.ts:6]: @alex-deaconu I didn't downgrade this one as I faced some problems when trying to get a dynamic value for the attribute directive. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu I had the base of this PR from our initial investigation of migrating controllers. Given is was halfway done, I decided to finalize it. Please review. Thanks!

File: src/app/components/focus-me/focus-me.directive.ts (hunk 2 above)
Comment (alex-deaconu) [reply src/app/components/focus-me/focus-me.directive.ts:6]: @ezequielc sounds good. I'm just wondering if this should go into `components/` or just `shared/directives`. Thanks.

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good 👍

**Pull request merged**